runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/users/register/view.ts

```diff
@@ -0,0 +1,13 @@
+import englishViewRegisterTranslations from '#src-app/translations/en/users/register/view';
+
+const viewRegisterTranslations: typeof englishViewRegisterTranslations = {
+    'Users.Register.View.Events.Success.AcceptingUser': 'Sukces',
+    'Users.Register.View.Events.Error.RolesNotSelected': 'Nie wybrano wszystkich wymaganych roli',
+    'Users.Register.View.Events.Error.AcceptFailed': 'Nie można zaakceptować wybranych użytkowników',
+    'Users.Register.View.Events.Error.FindingUsers': 'Wystąpił błąd podczas znajdowania nie aktywowanych użytkowników',
+    'Users.Register.View.SearchBarPlaceholder': 'Wyszukaj',
```

Comment:
on the blog there's "szukaj" 
<img width="960" alt="image" src="https://github.com/runbotics/runbotics/assets/69820069/8707c9a2-d9b7-45f8-b295-95bc881bec17">

stick to it